index.js

```diff
@@ -0,0 +1,53 @@
+const list = ['掃除', '買い物', '散歩'];
+const display_text = `現在持っているタスク一覧\n`;
+function list_display() {

```

Comment:
javascript では `snake_case` よりも `camelCase` が利用されることが多いです。

```suggestion
function listDisplay() {
```